<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remember_game</class>
 <widget class="QMainWindow" name="Remember_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : gray</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>900</width>
      <height>500</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : white</string>
      </property>
      <property name="text">
       <string>게임 시작 </string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="level">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : white</string>
      </property>
      <property name="text">
       <string>난이도 조작 </string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : lightgray; font : 15pt ; color : black;  </string>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="size">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size :50pt; background-color: white</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="KeyEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color : white</string>
      </property>
      <property name="text">
       <string>가로(=세로) 키의 수</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>391</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="Answer">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>391</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="Question">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
      </layout>
     </widget>
     <widget class="QPushButton" name="back_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : lightgray; font : 15pt ; color : black;  </string>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="restart">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : white</string>
      </property>
      <property name="text">
       <string>게임 재시작</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="toMain">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : white</string>
      </property>
      <property name="text">
       <string>메인 화면으로 돌아가기</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="score">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>SCORE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_score">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
